app/controllers/api/dance_directions_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Api::DanceDirectionsController < ApplicationController
+  def create
+    @dance_direction = DanceDirection.new(dance_direction_params)
+    @dance_direction.save
+
+    dance_direction = [@dance_direction]
+
+    render json: dance_direction
```

Comment:
Since you are serving a single 'record' as JSON, it is more typical to have the JSON be an object instead of an array:

```
render json: { dance_direction: @dance_direction }
```

For this, you are probably going to want a little bit of a control flow for if the dance direction cannot be saved:

```
if @dance_direction.save
  render json: { dance_direction: @dance_direction }, status: 201
else
 render json: { errors: @dance.direction.errors.full_messages }, status: 422
end
```